src/components/Filters.jsx

```diff
@@ -0,0 +1,58 @@
+export default function Filters(){
+    return(<aside>
+            <div class="filter">
+                <div class="filter-and-clear">
+                    <div>Filters</div>
+                    <div>
+                        <u>Clear</u>
+                    </div>
+                </div>
+                <form action="">
+                    <div class="input price">
+                        <label class="input-label" for="price">Price</label>
+                        <input class="input-data" type="range" id="price" name="price" min="0" max="100"></input>
+                        <div class="price-values">
+                            <div>0</div>
+                            <div>50</div>
+                            <div>100</div>
+                        </div>
+                    </div>
+                    <p class="filter-heading">Category</p>
+                    <div class="input inline-input">
+                        <input type="checkbox" id="men-clothing" name="men-clothing" checked></input>
```

Comment:
Remove "checked", if you don't want to give the value.

Reply:
Sure will remove it in later commits